.gitignore

```diff
@@ -1,5 +1,6 @@
 *~
 node_modules/
+logs/
```

Comment:
Shouldn't we leave this versioned with a `.keep` file in it?

Reply:
Sorry i don't know, how to do that , you can ajust it and add some update

Reply:
I mean adding `logs/` to git.

```sh
mkdir logs
touch logs/.keep
git add logs/.keep
```

And remove `logs/` from `.gitignore`.

Reply:
ok, i can do it

Reply:
You can check it in the new commits

Reply:
I'm on it. Thanks :v: